workspaces/announcements/.changeset/strong-flowers-remain.md

```diff
@@ -0,0 +1,4 @@
+---
+---
+
+Adds a minimal backstage instance with the announcements plugins installed. It is intended to be used for testing and development purposes, along with demonstrating all the features of the announcements plugin.
```

Comment:
If the changes don't impact a published package you simple don't add a changeset.

Now you did update the `README` in a few of the plugins so those should get a changeset(s) so that those changes get reflected on NPM 👍

Reply:
fixed - thank you.